### PR TITLE
[docs] add example for utils throttle

### PR DESCRIPTION
**Pull request by dirkk0**
demo here:
http://curious-electric.com/aframe/throttle/

**Description:**

**Changes proposed:**
-
-
-

### CODE REVIEW

File: docs/core/utils.md [hunk 1]
```diff
@@ -154,13 +154,27 @@ Checks if device is a smartphone. Returns a `boolean`.
...
 
-### `AFRAME.utils.throttle (function, interval [, optionalContext])`
+### `AFRAME.utils.throttle (function, minimumInterval [, optionalContext])`
 
 Returns a throttled function that is called at most once every
 `minimumInterval` milliseconds. A context such as `this` can be provided to
 handle function binding for convenience.
 
-### `AFRAME.utils.throttleTick (function (t, dt) {...}, interval [, optionalContext])`
+
```
Comment (ngokevin) [inline docs/core/utils.md]: big empty line here

File: docs/core/utils.md [hunk 2]
```diff
@@ -154,13 +154,27 @@ Checks if device is a smartphone. Returns a `boolean`.
...
+```js
+AFRAME.registerComponent('foo', {
```
Comment (ngokevin) [inline docs/core/utils.md]: `throttle` should be used on non-tick functions. `throttleTick` should be used for `tick` functions.

**Comment by ngokevin** [comment]
It's not quite correct, I would not use `throttle` when `throttleTick` should be used. Throttling `tick` shouldn't be explained in the `throttle` section. I don't think `throttle` needs to an example, `throttleTick` could have one though.

**Comment by dirkk0** [comment]
I don't quite understand? What would I use if I need to fire something
every 1/10 second and still need the regular tick function?
As far as I understood (and I tested it) with throttleTick the regular tick
gets replaced, at least in the given example.

**Comment by dirkk0** [comment]
ok, hopefully this works.

**Pull request merged**